include/ur_client_library/ur/ur_driver.h

```diff
@@ -180,6 +181,29 @@ class UrDriver
    */
   bool writeJointCommand(const vector6d_t& values, const comm::ControlMode control_mode);
 
+  /*!
+   * \brief Writes a trajectory point onto the dedicated socket.
+   *
+   * \param values Desired joint or cartesian positions
+   * \param goal_time Time for the robot to reach this point
+   * \param cartesian True, if the point sent is cartesian, false if joint-based
```

Comment:
```suggestion
   * \param cartesian True, if the point sent is cartesian, false if joint-based
   * \param goal_time Time for the robot to reach this point
   * \param blend_radius  The radius to be used for blending between control points
```
Included reordering to fit order of parameters in function definition.